File: examples/py/properties/properties.py
#!/usr/bin/env python

import signal
import gi

gi.require_version("Fin", "0.2")

from gi.repository import Fin

fin = Fin.Client.new()

print("Revision (property):", fin.props.revision)

print("Revision (method):", fin.get_revision())

print("EEPROM (property):", fin.props.eeprom)

print("EEPROM (method):", fin.get_eeprom())

print("UID (property):", fin.props.uid)

print("UID (method):", fin.get_uid())

signal.pause()

File: examples/js/properties/properties.js
#!/usr/bin/env node

const gi = require('node-gtk')
const sleep = require('sleep')

Fin = gi.require('Fin', '0.2')

const fin = new Fin.Client()

console.log("Revision:", fin.revision)

console.log("EEPROM:", fin.eeprom)

console.log("UID:", fin.uid)

while (true) {
    sleep.sleep(10);
}
